config/deploy/staging.rb

```diff
@@ -0,0 +1,61 @@
+# server-based syntax
```

Comment:
This file only contains comments, do we need it?

Reply:
It was auto-generated during the workshop. Let's not take any risks and assume this comment is vital.